docs/development.md

```diff
@@ -0,0 +1,63 @@
+# Development
+
+## Installing
+
+1. Install dependencies
+    - Rust
+    - Rocks DB ?
+    - FlatBuffers ?
```

Comment:
@mmartinbar @mariocao please shout out :wink:

Reply:
`Flatbuffers` is a dependency in `data-structures` and it is not necessarily a prerequisite for developers. `Flatbuffers` may be required only if devs want to compile and re-generate the `fbs` files.